v3/lints/etsi/lint_qcstatem_qctype_web_test.go

```diff
@@ -28,7 +28,8 @@ func TestEtsiQcTypeWeb(t *testing.T) {
 		"QcStmtEtsiNoQcStatmentsCert22.pem": lint.NA,
 		"qcSmimeNatural.pem":                lint.NA,
 		"qcSmimeLegal.pem":                  lint.NA,
-		"qcLegal.pem":                       lint.Error,
+		"qcLegal.pem":                       lint.NA,
```

Comment:
This one stopped reaching its desired error because it did not have the server auth EKU.